### PR TITLE
Missed PR fixes

### PR DESCRIPTION
**Pull request by afloresOSI**

### CODE REVIEW

File: content/add-organize-data/collect-data/connectors/pi-to-ocs/pi-to-ocs-data-transfers/landing-page3.md [hunk 1]
```diff
@@ -2,11 +2,10 @@
 uid: landing-page3
 ---
 
+# View PI system data for transfer
```
Comment (rmuolic) [inline content/add-organize-data/collect-data/connectors/pi-to-ocs/pi-to-ocs-data-transfers/landing-page3.md:5]: I think it should be "View PI System data for transfer"?

**Review by rmuolic — APPROVED** [review]
Reviewed and also committed a few updates.

**Review by VictoriaTouati-AVEVA — APPROVED** [review]
Reviewed & approved.

**Review by VictoriaTouati-AVEVA — CHANGES_REQUESTED** [review]
The topic title change to "View PI System data for transfer" should be "Transfer PI System data" because the main point of this procedure is to illustrate how to transfer data to OCS after querying for and selecting the PI points for the transfer. I will change the topic title in the procedure, menu, and landing page on Angela's branch.

**Review by VictoriaTouati-AVEVA — COMMENTED** [review]
@rmuolic & @afloresOSI: I changed the title "View PI System data for transfer"  to "Transfer PI System data to OCS" to make it technically accurate.

**Comment by rmuolic** [comment]
@afloresOSI this PR looks okay now to close to merge.

**Pull request merged**